src/ui/map.js

```diff
@@ -295,6 +295,8 @@ const defaultOptions = {
  *  * [Albers](https://en.wikipedia.org/wiki/Albers_projection) equal-area conic projection as `albers`
  *  * [Equal Earth](https://en.wikipedia.org/wiki/Equal_Earth_projection) equal-area pseudocylindrical projection as `equalEarth`
  *  * [Equirectangular](https://en.wikipedia.org/wiki/Equirectangular_projection) (Plate Carrée/WGS84) as `equirectangular`
+ *  * Globe, a 3D spherical projection, as `globe`
```

Comment:
Also, the glossary link from ProjectionSpecification above seems arcane, can we link to the style spec page instead? https://docs.mapbox.com/mapbox-gl-js/style-spec/projection/